analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/DisablingRequestValidationBase.cs

```diff
@@ -69,5 +80,79 @@ protected DisablingRequestValidationBase(System.Resources.ResourceManager rspecR
                 },
                 SymbolKind.NamedType,
                 SymbolKind.Method);
+
+            context.RegisterCompilationAction(c =>
+            {
+                analyzerConfiguration.Initialize(c.Options);
+                if (!analyzerConfiguration.IsEnabled(DiagnosticId))
+                {
+                    return;
+                }
+                foreach (var fullPath in Directory.GetFiles(rootPath, "web.config", SearchOption.AllDirectories).Select(p => Path.GetFullPath(p)))
```

Comment:
I would extract `Directory.GetFiles(rootPath, "web.config", SearchOption.AllDirectories).Select(p => Path.GetFullPath(p))` to a method.